download/download.go

```diff
@@ -510,3 +507,30 @@ func GetURLsFile(urlsFilePath string) (urlsList []string, err error) {
 
 	return urlsList, scanner.Err()
 }
+
+func AnonymizeFilepath(filePath string) string {
+	filePathSplit := strings.Split(filePath, "/")
+	if strings.Contains(filePathSplit[0], "_") {
+		_, err := mail.ParseAddress(strings.ReplaceAll(filePathSplit[0], "_", "@"))
+		if err == nil {
+			filePath = strings.Join(filePathSplit[1:], "/")
+		}
+	}
+
+	return filePath
+}
+
+func getPublicKey64() (string, error) {
```

Comment:
This function looks quite generic, so maybe better to be placed in `helpers.go`.

Reply:
Fixed in 3a00170